content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+---
```

Comment:
This is part of the PTFE /admin APIs, which @davidcelis has mostly documented in a PR in the private copy of the repo. Since they're all in the same family, we should adjust this page so it lines up with the other ones! That means: 

- It should go in a subdirectory of `docs/enterprise/api`, at `docs/enterprise/api/admin`.
- Its filename should be `settings.html.md`.
- Whenever it links to other pages with a relative path, there should be an additional `../` at the front.
- It should get a nav sidebar link (source/content/layouts/enterprise2.erb) in that new admin subsection of the API links. See David's PR for example; this'll make a merge conflict once we merge both, but meh, that's easy to deal with.